docs/static/docs.css

```diff
@@ -31,6 +31,10 @@ span.hljs-meta {
   color: #66d9ef;
 }
 
+a {
+  text-decoration: none;
+}
+
```

Comment:
This override is very broad, if we need to remove the underline e.g. for the side navigation then I would prefer to do something more targeted / specific. It's also not clear to me that it's related to the main changes being made in this PR?

Reply:
```suggestion
```

Reply:
This was an accidental addition - I'll remove it.

Reply:
Just FYI - this is a broad change in Bootstrap 5:
> Links are now underlined by default (not just on hover), unless they’re part of specific components.

Do we want to remove the underline by default in some  components so it doesn't have to be removed manually like in the [NavBar snippet](https://github.com/facultyai/dash-bootstrap-components/blob/v1/docs/components_page/components/navbar/navbar.py#L35) ?

Reply:
It’s such an annoying change…

I think @tcbegley broadly wants the underline removed, but didn’t like how broad this approach was as it could case the snippets to behave differently in the docs, so instead to remove on a case by case basis. Although I would definitely argue that things like the NavBar should have all links without the underline by default?